SinaiStCatherine/SinaiEt004.xml

```diff
@@ -521,7 +546,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                       <persName ref="PRS13957Qalementos"/>.</desc>
                                     <q xml:lang="gez">ወዓእረፈ፡ ቀሲስ፡ ጴጥሮስ፡ አሡሩ፡ ወረቡዑ፡ ለግንቦት፡ እምተሱኡ፡ በሮሜ። 
                                       ፷፻ወበ፱፻፡ ፶ወ፪፡ ለአብነ፡ ዓዳም፡ ወጸሐፎ፡ ኃጥእ፡ ዝልጉስ፡  በስም፡ መነኮስ፡ <persName ref="PRS13957Qalementos">ቀሌምንጦስ</persName>።</q> 
-                                    <note>Cp. <ref type="mss" corresp="SinaiEt006_#a4"/>.</note>                                    
+                                    <note>Cp. <ref type="mss" corresp="SinaiEt006#a4"/>.</note>                                    
```

Comment:
if you want the shelfmark to be printed it must be either 

`<ref type="mss" corresp="SinaiEt006"/>, <ref corresp="SinaiEt006#a4">additio 4</ref>`

or 

`<ref type="mss" corresp="SinaiEt006#a4">Sinai Eth. 4 additio 4</ref>`

otherwise the app would not know what to do with this link